src/components/carousel/index.jsx

```diff
@@ -232,7 +264,6 @@ Carousel.Item = Item;
 Carousel.defaultProps = {
 	pageCountPhrase: () => {},
 	showLabel: false,
-	slidesToShow: 4,
```

Comment:
looks like this default affected the changed tests https://github.com/WPMedia/arc-themes-components/pull/75/files#r853404822

Reply:
`screen.queryByRole("button", { name: "Next" })` does grab the appropriate button.